packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -104,6 +99,14 @@ const getSwitchStyles = (theme: GrafanaTheme2, transparent?: boolean) => ({
         },
       },
 
+      '&:disabled + label': {
```

Comment:
Before the change, when `Switch` was `checked and disabled`, the disabled style was **overridden** by the checked (L88)

Reply:
I finally reverted this change as the frontend team asked for a separate PR for this.